source/time_tables/departure_boards.h

```diff
@@ -58,6 +63,17 @@ time_duration length_of_time(const time_duration& duration_1,
 bool line_closed (const time_duration& duration,
                   const time_duration& opening,
                   const time_duration& closing,
-                  const pt::ptime& date );
+                  const boost::posix_time::ptime& date );
+
+std::pair<DateTime, DateTime> shift_next_stop_time_to_opening_time(const routing::datetime_stop_time& next_stop_time,
+                                                                   const uint32_t opening_time);
```

Comment:
pt = posix::time but in this header file pt is not declared.

As said @kinnou02  https://github.com/CanalTP/navitia/pull/2509#discussion_r214627329

Reply:
The point from kinnou is to avoid aliases in header, not to avoid using the type, in my understanding.
So to me it would be nicer to have a posix::time::time_duration.

Reply:
I'll try to change that if you want for more reading confort.